Tests/ManualTests/ManualTestingApp/Singles/VideoPage.ux

```diff
@@ -3,47 +3,49 @@
 		<p>This page contains a video-player, that should show the video "Big Buck Bunny".</p>
 	</InfoStack>
 	<DockPanel>
-		<Panel>
-			<Panel ux:Name="_loadingMessage" Background="#000" Opacity="0.5" Visibility="Hidden">
-				<Text TextColor="#fff" Alignment="Center">Loading...</Text>
-			</Panel>
+		<DockPanel>
```

Comment:
Do we really need two nested dockpanels? Can't we just dock to top and bottom of the same one instead?

Reply:
ok, got rid of the extra dockpanel now :)